package.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "typeahead",
-  "version": "0.0.10",
+  "version": "1.0.0",
   "description": "",
   "main": "src/typeahead.js",
   "scripts": {
     "test": "karma start karma.conf.js"
   },
-  "author": "",
+  "author": "SOFTEC AG",
   "license": "ISC",
```

Comment:
Why licence ISC?

Reply:
normally we use MIT afaik

Reply:
I think it's in there because it's the default. We should change it to something else. What do you propose?